src/Api/WebClient.php

```diff
@@ -317,9 +324,14 @@ private function isCashOnDelivery(OrderInterface $order): bool
         return false;
     }
 
-    private function createParcel(ShipmentInterface $shipment): array
-    {
+    private function createParcel(
+        ShipmentInterface $shipment,
+    ): array {
         $weight = $shipment->getShippingWeight();
+        $template = 'large';
```

Comment:
I think we should put the value into container parameter, as it should be configurable.

Reply:
Done 👍 
Added ability to select default parcel_template and label_type to .env file.